static/js/src/advantage/subscribe/react/components/FreeTrialRadio.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { RadioInput, Row, Col } from "@canonical/react-components";
+import { formatter } from "../../renderers/form-renderer";
+import { add, format } from "date-fns";
+import usePreview from "../APICalls/usePreview";
+
+const DATE_FORMAT = "dd MMMM yyyy";
+
+type Props = {
+  isUsingFreeTrial: boolean;
+  setIsUsingFreeTrial: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const FreeTrialRadio = ({ isUsingFreeTrial, setIsUsingFreeTrial }: Props) => {
```

Comment:
Could you add some tests for this?

You can look at the one I did recently for ideas: https://github.com/canonical-web-and-design/ubuntu.com/blob/ea5f3f9f7ebca1162b8d99b10b747b4ad46fe33e/static/js/src/advantage/users/AccountUsers.test.tsx